environments/production/terraform.yml

```diff
@@ -516,6 +516,21 @@ rds_instances:
       wal_buffers: 32MB
       shared_buffers: 12GB
 
+  - identifier: "pgauditcare1-production"
+    instance_type: "db.t3.2xlarge"
+    storage: 6000
+    multi_az: true
+    engine_version: "10.17"
+    backup_window: "23:00-01:00"
+    backup_retention: 7
+    enable_cross_region_backup: true
```

Comment:
To save some costs during the switchover, can you set this to `false` to begin with, and then the day before the switchover, set it to `true`? Unless we think the total amount of time both will be up is just a few days

Reply:
@dannyroberts I'm planning to launch the new instance before a couple of hours of the maintenance window.